packages/grafana-schema/src/schema/graph.gen.ts

```diff
@@ -274,23 +278,27 @@ export interface VizLegendOptions {
   sortDesc?: boolean;
 }
 
+export const defaultVizLegendOptions: Partial<VizLegendOptions> = {
+  calcs: [],
+};
+
 export enum BarGaugeDisplayMode {
   Basic = 'basic',
   Gradient = 'gradient',
   Lcd = 'lcd',
 }
 
 export interface TableFieldOptions {
-  align: string;
+  align: FieldTextAlignment;
```

Comment:
I changed this on the TS side, rather than the CUE side, because CUE actually seemed more correct. Can change it back to just string, though.